packages/@aws-cdk/aws-eks/test/integ.eks-cluster.defaults.expected.json

```diff
@@ -922,6 +922,78 @@
         "ClusterResourceHandlerServiceRole7FB16465"
```

Comment:
If the helm chart handler will be created lazily, this file and all other .expected.json files should be reverted

Reply:
Reverted all changes to the `...exepected.json` files. A new file will be added with the integration test.